### DISCUSSION TITLE
Draft release not visible after deleting tag

### DISCUSSION THREAD

**Discussion by philips77** [community/community#7008] [Other Feature Feedback, Questions, & Ideas] [answered]
I created a release on GitHub project and realized i didn't push all commits.
I deleted the tag from local and remote.
It used to be, that in such case on GitHub appeared an unpublished release, waiting to be tagged. Not anymore. It's there, but now visible.
I have to create a new draft release, I type the same tag name (which the drop down displays as Create new tag (correctly)), and an option to restore the unpublished release shows up. But after doing that, it incorrectly shows that this tag already exists. Therefore i can't create a proper release anymore.

Ref: https://github.com/NordicSemiconductor/Android-DFU-Library/releases/tag/1.12.0
1. There is no tag 1.12.0
2. This release doesn't show up in Releases
3. I can't update it, as I get an error
```
We weren't able to create the release for you. Make sure you have a valid tag.
```

**Comment by philips77** [comment]
Ok, I had to delete the previously saved draft and create a new one again. That solved my problem, but still the issue exists.

**Comment by MylesBorins** [comment, marked as answer]
Hey @philips77. I just attempted to reproduce the flow you talked about. I had a release on GitHub, deleted the associated tag, and the release I had created converted to a draft. I pushed the tag again and was able to publish the release.

Are you able to reproduce this bug? If so could you make details reproduction steps?

**Reply by MylesBorins** [reply]
I'm going to mark this as answered, but please feel free to keep the conversation going
